### PR TITLE
Provide ability to pass token on the client side

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thamos/lib.py [hunk 1]
```diff
@@ -75,6 +75,7 @@
 _RETRY_ON_ERROR_COUNT = int(os.getenv("THAMOS_RETRY_ON_ERROR_COUNT", 3))
 _RETRY_ON_ERROR_SLEEP = float(os.getenv("THAMOS_RETRY_ON_ERROR_SLEEP", 3.0))
 _THAMOS_TIMEOUT = int(os.getenv("THAMOS_TIMEOUT", 2000))
+_THAMOS_TOKEN = os.getenv("THAMOS_TOKEN")
```
Comment (pacospace) [inline thamos/lib.py:78]: isn't the token requested only by Kebechet thoth integration?

Comment (fridex) [reply thamos/lib.py:78]: Any user submitting requests to Thoth backend with a valid token. By providing it, an authenticated request is done - we support this feature on our client library side, that is thamos.

Comment (pacospace) [reply thamos/lib.py:78]: Ah right, we did not filter by Integration enum, sorry, all good :)

Comment (pacospace) [reply thamos/lib.py:78]: We need also the token for adviser right?

Comment (fridex) [reply thamos/lib.py:78]: Not sure what token you mean here.

Comment (pacospace) [reply thamos/lib.py:78]: Sorry I didn't see the method changed exactly. All good!

**Review by pacospace — APPROVED** [review]
lgtm!

**Pull request merged**